R/find_site_coords.R

```diff
@@ -0,0 +1,36 @@
+#' Pull the latitude and longitude of a site from ScienceBase
+#' 
+#' @import dplyr
+#' @importFrom dataRetrieval readNWISsite
+#' @param site_names a list of site names such as those returned from 
+#'   make_site_name()
+#' @param format character specifying the desired output format. normal is a 
+#'   human-readable, dplyr-friendly format. geoknife works well for geoknife and
+#'   excludes sites with incomplete coordinate data.
+#' @return a data.frame of
+#' @export
+#' @examples 
+#' # middle site has missing coords:
+#' find_site_coords(c("nwis_01467200","nwis_09327000","nwis_351111089512501")) 
+#' find_site_coords(c("nwis_01467200","nwis_09327000","nwis_351111089512501"), format="geoknife")
+find_site_coords <- function(site_names, format=c("normal","geoknife")) {
```

Comment:
now that we're replacing most powstreams functionality, I want to vote once again we merge and drop powstreams